### PR TITLE
Avoid importing seer under Node.js

### LINKED ISSUE #394: Server Side Rendering creates global "window" object during import
Reopening https://github.com/uber/deck.gl/issues/1335 here.
CC: @ex3ndr @ibgreen 

Description from above issue:

Right after importing react-map-gl object "window"object "window" in the global scope appears and this breaks a lot of stuff on the server side.

```
console.warn(typeof window);
import 'deck.gl';
console.warn(typeof window);
```

Will output
```
undefined
deck.gl 5.0.2 - set deck.log.priority=1 (or higher) to trace attribute updates
object
```

Workaround is to use dynamic imports and do not import `deck.gl` directly.

### PR DESCRIPTION
**Pull request by ibgreen**
@balthazar Would be preferable to fix this directly in seer. 

Fixes #394

### CODE REVIEW

File: src/debug/seer-integration.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+const seer = isBrowser && require('seer');
 
 const models = {};
 
 /**
  * Add a model to our cache indexed by id
  */
 export const addModel = model => {
-  if (models[model.id]) {
+  if (!window.__SEER_INITIALIZED__ || models[model.id]) {
```
Comment (balthazar) [inline src/debug/seer-integration.js]: shouldn't do a check on `window` here too using `isBrowser`?

File: src/debug/seer-integration.js [hunk 2]
```diff
@@ -96,16 +97,18 @@ export const getOverrides = (id, uniforms) => {
...
+    }
+
+    const valuePath = payload.valuePath.slice(1);
+    setOverride(payload.itemKey, valuePath, payload.value);
+
+    const uniforms = model.getUniforms();
+    recursiveSet(uniforms, valuePath, payload.value);
+    model.setUniforms(uniforms);
+  });
+}
```
Comment (balthazar) [inline src/debug/seer-integration.js]: missing a newline here

File: src/debug/seer-integration.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
-import seer from 'seer';
-
 import {window} from '../utils/globals';
+import isBrowser from '../utils/is-browser';
+
+const seer = isBrowser && require('seer');
```
Comment (1chandu) [inline src/debug/seer-integration.js]: So this was modifying `window` , and now it is disabled when run under node?

**Comment by ibgreen** [comment]
Obsoleted by #396

**Pull request closed without merging**